internal/handlers/general/pagination.go

```diff
@@ -0,0 +1,52 @@
+package general
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/gofiber/fiber/v2"
+	"github.com/pilagod/gorm-cursor-paginator/v2/paginator"
+)
+
+const DefaultLimitCount = 10
+
+type PaginationLinks paginator.Cursor
+
+func NewPaginator(ctx *fiber.Ctx) *paginator.Paginator {
+	paginator := paginator.New(&paginator.Config{
+		Rules: []paginator.Rule{{Key: "ID"}, {Key: "CreatedAt"}},
+		Keys:  []string{},
+		Limit: DefaultLimitCount,
+		Order: paginator.ASC,
+	})
+
+	if after := ctx.Query("page[after]"); after != "" {
```

Comment:
Just for asking, why we need to query like that? Paginator doesn't do that by default?

Reply:
The name is maybe a bit misleading: that's not a database _query_ (the layer where paginator works), but a parameter from the _query string_ (from fiber)